### PR TITLE
FISH-651 FISH-868 FISH-1504 FISH-1526 FISH-1527 MicroProfile 4.0

### PR DESCRIPTION
**Pull request by Pandrex247**
Split off from https://github.com/payara/Payara-Enterprise-Documentation/pull/154

Title.
Main PR: https://github.com/payara/Payara-Enterprise/pull/407

Also brings across payara/Payara-Community-Documentation#115, since this seems to have been missed.
Also brings across payara/Payara-Community-Documentation#127, since this seems to have been missed.

### CODE REVIEW

**Review by OndroMih — CHANGES_REQUESTED** [review]
I added some suggestions in comments and also created a PR https://github.com/Pandrex247/Payara-Enterprise-Documentation/pull/2 with some more suggestions.

File: docs/modules/ROOT/pages/documentation/microprofile/healthcheck.adoc [hunk 1]
```diff
@@ -34,12 +34,12 @@ The MicroProfile Healthchecks API defines the following list of REST endpoints t
 |On invocation, all registered health checks annotated with `@Readiness` are performed and the results aggregated.
 
 |`/health`
```
[the remark this answers is not in the document]
Comment (Pandrex247) [reply docs/modules/ROOT/pages/documentation/microprofile/healthcheck.adoc:36]: It already has that?

File: docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc [hunk 2]
```diff
@@ -96,21 +90,23 @@ Provides a way to set the configuration of the fault tolerance service of the ta
 
 ===== Command Options
 
+NOTE: Since Payara Platform 5.29.0, the options `asyncmaxpoolsize` and `delaymaxpoolsize` have been removed as they are no longer applicable due to the configurable managed executor pools being used.
+
 [cols="4,10,1,1", options="header"]
```
[the remark this answers is not in the document]
Comment (Pandrex247) [reply docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc]: Already has that?

**Comment by Pandrex247** [comment]
Added additional docs pertaining to upgrade issues.

**Review by AlanRoth — APPROVED** [review]
Looks good, however, like my comment on #154, please remove any 'Since Payara' tags you see on pages you edited, the tag is deprecated on any pages going forward

**Pull request merged**